dotcom-rendering/src/components/GroupedNewsletterList.tsx

```diff
@@ -0,0 +1,58 @@
+import { css } from '@emotion/react';
+import {
+	headlineObjectStyles,
+	palette,
+	space,
+} from '@guardian/source-foundations';
+import type { GroupedNewsletters } from '../types/newslettersPage';
+import { NewsletterDetail } from './NewsletterDetail';
+import { Section } from './Section';
+
+export interface NewslettersListProps {
+	groupedNewsletters: GroupedNewsletters;
+}
+
+const groupContainerStyle = css`
+	display: flex;
+	flex-wrap: wrap;
+`;
+
+const groupItemStyle = css`
+	flex-basis: ${200}px;
+	margin-right: ${space[3]}px;
+	margin-bottom: ${space[3]}px;
+	padding: ${space[1] / 2}px;
```

Comment:
Interesting! I haven't seen this way of using Source in DCR but I suppose its a perfectly good way of doing it!